packages/react-components/src/organisms/SharedResearchCard.tsx

```diff
@@ -55,6 +63,8 @@ const SharedResearchCard: React.FC<SharedResearchCardProps> = ({
   published,
   publishingEntity,
   isInReview,
+  keywords,
+  showTags = false,
```

Comment:
Don't we want it to be true or call this component with showTags={true}? The tags are not appearing right now